### PR TITLE
SDK-225 : Integrate .env file into example

### PR DESCRIPTION
**Pull request by sidibos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,10 +170,12 @@ Allows your app to retrieve a user profile, given an encrypted token.
...
 * Create your application in the Yoti Dashboard (this requires having a Yoti account)
-* Fill in the YOTI_APPLICATION_ID and YOTI_SCENARIO_ID values in the index.html file
-* Fill in the SDK_ID and pem file path config details in the profile.php file
 * Point your Yoti application callback URL to `http://your-local-url.domain/profile.php`
-* Run `composer update` command inside [example folder](https://github.com/getyoti/php/tree/master/example)
+* Do the steps below inside the [example folder](https://github.com/getyoti/php/tree/master/example)
+* Copy `.env.dist` to `.env`
+* Open `.env` file and fill in the environment variables `YOTI_APPLICATION_ID`, `YOTI_SCENARIO_ID`, `YOTI_SDK_ID`, and `YOTI_KEY_FILE_PATH`
+* Run `composer update` command
+* Run `php -S localhost:8000` command and navigate to `http://localhost:8000`
```
Comment (vassyz) [inline README.md]: For these two lines, I feel like it sounds better if you use `the ... command` or remove the word `command`. Also, you can set `http://localhost:8000` as a URL `[http://localhost:8000](http://localhost:8000)`.

File: example/.env.dist [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# This file is a "template" of which env vars needs to be defined in your configuration or in an .env file
```
Comment (vassyz) [inline example/.env.dist]: This file is a template for defining the environment variables.

File: example/.env.dist [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+# Set the application config values here
+
+YOTI_APP_ID=xxxxxxxxxxxxxxxxxx # This is your application ID
+YOTI_SCENARIO_ID=xxxxxxxxxxxxxxxx # This is the scenario ID
+YOTI_SDK_ID=xxxxxxxxxxxxxxxxxxxxx # This is your SDK ID associated with the Yoti Application you created on Dashboard
```
Comment (vassyz) [inline example/.env.dist]: Remove these three comments. It's pretty clear what the variables represent.

**Comment by vassyz** [comment]
Apart from some minor text amends everything looks good.

**Comment by sidibos** [comment]
Used quote around template because a template could also be a html view template. Anyway have removed it.

**Pull request merged**